src/interface.js

```diff
@@ -0,0 +1,122 @@
+const guardianAPIKey = 'INSERT_YOUR_API_KEY_HERE';
+const guardianUrl = 'https://content.guardianapis.com/search?show-fields=all&api-key='+ guardianAPIKey;
+const aylienurl = 'http://news-summary-api.herokuapp.com/aylien?apiRequestUrl=https://api.aylien.com/api/v1/summarize?url=';
+const newsCollection = new NewsCollection();
+const date = new Date();
+const d = new Intl.DateTimeFormat('en-US').format(date);
+document.getElementById('date').innerHTML = d;
+
+
+window.onload = function() {
+  hideContent('full_news');
+  getNews();
+};
+
+
+window.onhashchange = function() {
+  const newsUrl = getUrl();
+  if (newsUrl == null || !newsUrl) {
+    getNews();
+  } else {
+    getSummary(newsUrl);
+  }
+};
+
+function getSummary(newsUrl) {
+  hideContent('news_collection');
+  summarizeText(newsUrl);
+}
+
+function getUrl() {
+  return window.location.hash.split('#')[1];
+}
+
+
+function getNews() {
+  const requestUrl = guardianUrl;
+  hideContent('news_summary');
+  fetch(requestUrl, {
+    method: 'GET',
```

Comment:
I'm not sure if it is essential to include lines 39-42 as I think they are the default options. However, it doesn't make it less readable to include them